### PR TITLE
[FIX] base: assets fix on windows

### PR DESCRIPTION
**Pull request by brboi**
Description of the issue/feature this PR addresses:
Assets cannot load when Odoo server runs on Windows

Current behavior before PR:
Assets cannot load when Odoo server runs on Windows

Desired behavior after PR is merged:
Assets can load when Odoo server runs on Windows

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by brboi** [comment]
@odoo/rd-security

File: odoo/addons/base/models/ir_asset.py [hunk 1]
```diff
@@ -348,7 +348,7 @@ def is_safe_path(path):
                     return False
                 if path.rpartition('.')[2] in TEMPLATE_EXTENSIONS:
                     # Forbid xml to leak
-                    return ("%s/static/" % addon) in path
+                    return (os.path.normpath("%s/static" % addon) + os.path.sep) in path
```
Comment (xmo-odoo) [inline odoo/addons/base/models/ir_asset.py:351]: This should have a comment explaining that `normpath` will strip the trailing `/`, which is why it has to be added afterwards.

File: odoo/addons/base/models/ir_asset.py [hunk 2]
```diff
@@ -359,7 +359,7 @@ def is_safe_path(path):
             # files are read from the file system. But web assets (scripts and
             # stylesheets) must be loaded using relative paths, hence the trimming
             # for non-xml file paths.
-            paths = [path if path.split('.')[-1] in TEMPLATE_EXTENSIONS else path[len(addons_path):] for path in paths]
+            paths = [path if path.split('.')[-1] in TEMPLATE_EXTENSIONS else fs2web(path[len(addons_path):]) for path in paths]
```
Comment (xmo-odoo) [inline odoo/addons/base/models/ir_asset.py:362]: Shouldn't the path be fs2web'd in both cases here?

Comment (brboi) [reply odoo/addons/base/models/ir_asset.py:362]: Template files are read from the file system, so their path should not be fs2web´d

**Comment by brboi** [comment]
@xmo-odoo I took your comments into account.

**Comment by brboi** [comment]
@odoo/rd-security can you review again this PR ?
Thanks

**Comment by xmo-odoo** [comment]
@robodoo override=ci/security

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**